app/scripts/modules/kubernetes/src/container/ports.component.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.deck.kubernetes.container.ports.component', [])
```

Comment:
I think some modules are named `spinnaker.deck.kubernetes...`, but they should just be `spinnaker.kubernetes...`.